src/sentry/static/sentry/app/views/alerts/wizard/index.tsx

```diff
@@ -99,25 +101,31 @@ class AlertWizard extends React.Component<Props, State> {
             <StyledPageHeader>
               <PageHeading>{t('What do you want to alert on?')}</PageHeading>
             </StyledPageHeader>
+            <Heading>{t('Errors')}</Heading>
```

Comment:
had to move the `Errors` heading above so that the wizard panel would be in line with the first option in the wizard